about.html

```diff
@@ -0,0 +1,99 @@
+            </ul>
+        </nav>
+    </header>
+    <section class="main about-cover" id="background">
+        <div class="about-mid">
+            <h1 class="about-h1">Hello Piano world!</h1>
+            <h2 class="about-header">Piano CONCERT 2023</h2>
+            <p class="invitation-about">
+                Piano concert invition.Welcome to our concert. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quidem, veniam enim? Corporis quisquam quos obcaecati minus consectetur ratione iusto veritatis nam, illo quibusdam itaque aut, dolorum unde culpa
+                delectus. Necessitatibus totam, esse delectus doloribus repellendus enim illum exercitationem rerum dolorem dolor voluptate amet? Placeat adipisci ducimus, molestias illum aperiam provident?</p>
+
+            <p class="contact-us">
+                Do you have questions about our concert Event? </p>
+            <a href="#" class="email">hsuwady2001@gmail.com </a>
+        </div>
+    </section>
+    <section class="section-2">
+        <h2 class="about-h2">Piano Concert 2023 Event</h2>
+        <div class="bottom-line"></div>
+        <p class="  about-sponser">
+            Delve into the captivating stories from our previous summer concerts held in diverse locations. Experience the mesmerizing melodies that filled the air, creating an enchanting ambiance for all attendees.
+            <br><br> These extraordinary events showcased the power of music and unity within our own local community. Immerse yourself in the inspiring narratives of these memorable gatherings, where the spirit of music and the joy of cultural exchange
+            seamlessly intertwined.
+        </p>
+        <img src="images/piano logo.png" alt="logo-2">
+    </section>
+
+    <section class="section-3">
+        <h2 class="about-h2">Discover previous piano concerts</h2>
+        <div class="bottom-line"></div>
+        <p class="about-sponser">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatum sint quas fuga dignissimos expedita impedit possimus deserunt hic in facere?
+            <br><br> Lorem ipsum dolor, sit amet consectetur adipisicing elit. Repellendus sapiente facilis beatae praesentium quia maiores! Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nisi nesciunt et delectus, doloremque minus perspiciatis.
+        </p>
+
+        <div class="past-projects">
+            <div class="project-1 project">
+            </div>
+            <div class="project-2 project">
+            </div>
+        </div>
+    </section>
+    <footer class="footer">
+        <img class="logo" src="./images/piano logo.png" alt="logo ">
+        <div class="footer-text ">
+            <span class="footer-copyright ">2023  Piano concert.Rights Reserved. </span>
+            <span class="footer-copyright ">사랑하는 친구에게,
+오랜만에 펜을 들어 글을 쓰는 순간, 당신이 떠올랐습니다. 얼마나 그리웠던지 몰라요. 지금은 멀리 떨어져 있지만
+ </span>
+        </div>
+    </footer>
```

Comment:
- [ ] Please note that the footer on the mobile version should be white. This is because we have to follow the colors used in the design. We are permitted to change the content (text & images) but to keep the rest the same. 😄 

![MOB-FOOTER](https://github.com/Hsuwaddyhtat22/Microverse-first-capstone-project/assets/99656687/9ffff417-cadf-46e6-a898-351f890a61a7)

Reply:
- [ ] For the desktop size, note the followng differences between the footers and kindly implement them.
-  About Page: It has a dark background. 👍🏾 
- Home Page: It has a white background. 👍🏾